src/sentry/api/endpoints/monitor_checkin_details.py

```diff
@@ -76,7 +76,7 @@ def convert_args(self, request, monitor_id, checkin_id, *args, **kwargs):
     def get(self, request, project, monitor, checkin):
         """
         Retrieve a check-in
-        ``````````````````
+        ```````````````````
 
         :pparam string monitor_id: the id of the monitor.
         :pparam string checkin_id: the id of the check-in.
```

Comment:
I might be stupid..but...

Is this intentional? The function doesn't take those params. Also is "pparam" a typo?

Reply:
this is for our doc generator

Reply:
Gotcha.